### PR TITLE
Update Italian strings

### PR DESCRIPTION
**Pull request by bovirus**
Update Italian strings for preprocessor and program group icon descriuption

### CODE REVIEW

File: setup.iss [hunk 1]
```diff
@@ -167,12 +167,12 @@ Source: "files\{#isppdll}"; DestName: "ISPP.dll"; DestDir: "{app}"; Flags: ignor
...
-Name: "{group}\Inno Setup Compiler"; Filename: "{app}\Compil32.exe"; WorkingDir: "{app}"; AppUserModelID: "JR.InnoSetup.IDE.5"
-Name: "{group}\Inno Setup Documentation"; Filename: "{app}\ISetup.chm"
-Name: "{group}\Inno Setup Example Scripts"; Filename: "{app}\Examples\"
-Name: "{group}\Inno Setup FAQ"; Filename: "{app}\isfaq.htm"
-Name: "{group}\Inno Setup Revision History"; Filename: "{app}\whatsnew.htm"
-Name: "{commondesktop}\Inno Setup Compiler"; Filename: "{app}\Compil32.exe"; WorkingDir: "{app}"; AppUserModelID: "JR.InnoSetup.IDE.5"; Tasks: desktopicon
+Name: "{group}\{cm:InnoSetupCompiler}"; Filename: "{app}\Compil32.exe"; WorkingDir: "{app}"; AppUserModelID: "JR.InnoSetup.IDE.5"
+Name: "{group}\{cm:InnoSetupDocumentation}"; Filename: "{app}\ISetup.chm"
+Name: "{group}\{cm:InnoSetupExampleScripts}"; Filename: "{app}\Examples\"
+Name: "{group}\{cm:Inno Setup FAQ}"; Filename: "{app}\isfaq.htm"
```
Comment (martijnlaan) [inline setup.iss]: There are spaces here which shouldn't be there?

File: setup.iss [hunk 2]
```diff
@@ -187,6 +187,21 @@ ISPPSubtitle=Would you like to install Inno Setup Preprocessor?
...
+italian.ISPPSubtitle=Vuoi installare Inno Setup Preprocessor?
+italian.ISPPText=Inno Setup Preprocessor (ISPP) è un addon ufficiale per Inno Setup.%n%nISPP ti permette condizionalmente di compilare parte degli script, di usare variabili tempo durante la compilazione negli script e di usare le funzioni integrate come ad esempio leggere il registro o file INI durante la compilazione.%n%nISPP contiene inoltre una versione speciale del compilatore a linea di comando ISCC che accetta la definizione delle variabili come parametri da linea di comando e può usarle durante la compilazione.
+italian.ISPPText2=Seleziona se vuoi installare ISPP, quindi seleziona 'Avanti'.
+italian.ISPPCheck=&Installa Inno Setup Preprocessor
+InnoSetupCompiler=Inno Setup Compiler
+InnoSetupDocumentation=Inno Setup Documentation
+InnoSetupExampleScripts=Inno Setup Example Scripts
+InnoSetupFAQ=Inno Setup FAQ
+InnoSetupRevisionHistory=Inno Setup Revision History
+italian.InnoSetupCompiler=Compilatore Inno Setup
```
Comment (martijnlaan) [inline setup.iss]: I'm not sure what to think about this. The Compiler IDE isn't available in Italian so translating the icon for it seems a bit misleading?

Comment (bovirus) [reply setup.iss]: I believe that is not important that the program has an main English interface.
It's important to have icon description in Italian (localized).

Please take care that Inno Setup has an Italian installer (with preprocessor window in English - 
Now should be fixed).
The Inno Setup Quick Start Pack has an English installer.
Should be fine also to have Inno Setup Quick Start Pack with Italian language interface for the installer.
I sent all Italian strings/modification of iss installer script for Inno Script Studio standalone to the author.
I asked to the Inno Script Studio author 
- to include in Inno Setup Studio all languages available.
- to make separate folder for Inno Script Studio (under \Program Files x86\Kymoto Solutions\Inno Script Studio)
- to have Italian language for the installer
- to fix problem about OK/Cancel button in the windows that are untranslated.
 


I'm working also with the InnO Script Studio author to have the insttaler in italian.

**Comment by bovirus** [comment]
@martijnlaan 

Could you pelase implemnent multilanguage in ispack instalelr script

https://github.com/jrsoftware/ispack/blob/master/setup.iss

File: setup.iss (hunk 1 above)
Comment (bovirus) [reply setup.iss]: Sorry.
My errori. Should be

"{group}\{cm:InnoSetupFAQ}"; Filename: "{app}\isfaq.htm"

without spaces. Thanks.

Comment (martijnlaan) [reply setup.iss]: Yes, can you do a commit to fix it?

File: setup.iss (hunk 2 above)
Comment (martijnlaan) [reply setup.iss]: Ok

**Comment by martijnlaan** [comment]
Feel free to make a pull request on the ispack repo as well. I do think there's too many custom strings in that installer to add all languages, so you should just add Italian and not all.

File: setup.iss (hunk 1 above)
Comment (bovirus) [reply setup.iss]: Added a new pullup request

https://github.com/jrsoftware/issrc/pull/267

Thanks.

**Comment by martijnlaan** [comment]
See #267

**Pull request closed without merging**